src/app-pages/map/components/map-nav-bar/MapNavBar.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import PropTypes from 'prop-types';
+import SearchBox from "../../../../app-common/SearchBox";
+import DropDown from "../../../../app-common/Dropdown";
+import MenuTree from "../../../../app-common/tree-menu/TreeMenu";
+import "./mapNavBar.scss";
+import { connect } from "redux-bundler-react";
+
+const MapNavBar = ({
+  locationTree,
+  doSetSelectedLocationCode,
+}) => {
+  const [orgDivToggleState, setOrgDivToggleState] = useState(false);
+
+  const handleNodeClick = (e) => {
+    //if node is a leaf then toggle the drawer
+    if (!e.hasNodes) {
+      doSetSelectedLocationCode(e.location_id);
+    }
+  };
+
+  const handleOrgDivClick = () => {
+    setOrgDivToggleState(!orgDivToggleState);
+  };
+
+  return (
+    <div className="map-nav-bar-wrapper">
+      <div className="h-100 px-5 py-4 bg-secondary">
+        <div className="row map-nav-row">
+          <div className="col-md-4">
+            <SearchBox />
+          </div>
+          <div className="col-md-1">
+            <DropDown
+              label={"Districts Dropdown"}
+              id={"districts-dropdown"}
+              options={["Select District", "1", "2", "3"]}
+            />
+          </div>
+          <div className="col-md-1">
+            <DropDown
+              label={"Basin Dropdown"}
+              id={"basins-dropdown"}
+              options={["Select Basin", "1", "2", "3"]}
+            />
+          </div>
+          <div className="col-md-2">
+            <DropDown
+              label={"Location Type"}
+              id={"location-type-dropdown"}
+              options={[
+                "Location Type",
+                "Divisions",
+                "Districts",
+                "Projects",
+                "Stream Gages",
+                "Sites",
+                "WQ",
+              ]}
+            />
+          </div>
+          <div className="col-md-4">
+            <div
```

Comment:
Does the client care about accessibility? I noticed this div isn't reachable via keyboard. If the site has to be 508 compliant then we should make this a button (though the styling would have to be updated as well)
```
            <button
              type="button"
              className="organizational-structure-div"
              onClick={handleOrgDivClick}
            >
              Organizational Structure
            </button>
```

Reply:
Good point. I'll push up the fix.